logger/request.go

```diff
@@ -18,18 +20,13 @@ type header struct {
 
 // PartnerRequest contains information of requests we sent to our partners
 type PartnerRequest struct {
```

Comment:
@viettd Yes, just keep this `naming` for `compatibility` now(in case when I do the update of `payments`, I will mix the `Request` and `PartnerRequest`), I will update `payments` accordingly, after `payments` is updated, we can remove this actually.